src/download.ts

```diff
@@ -184,36 +154,27 @@ export async function getTranslators(targetLanguageIds: string[]): Promise<strin
 		blockedUsers.push(blockedUser.id);
 	}
 
-	const requests = [];
+	const topMembers = new Map<string, string[]>();
 	for (const languageId of targetLanguageIds) {
-		requests.push(
-			requestLimit(() =>
-				(async () => {
-					const { status: reportStatus, identifier: reportId } = (
-						await reportsApi.generateReport(PROJECT_ID, {
-							name: 'top-members',
-							schema: {
-								unit: 'words',
-								format: 'json',
-								dateFrom: '2014-01-01T00:00:00+00:00',
-								dateTo: '2030-01-01T00:00:00+00:00',
-								languageId
-							}
-						})
-					).data;
-					let finished = reportStatus === 'finished';
-					while (!finished) {
-						await wait(3000);
-						finished = (await reportsApi.checkReportStatus(PROJECT_ID, reportId)).data.status === 'finished';
-					}
-					return (await AXIOS.get((await reportsApi.downloadReport(PROJECT_ID, reportId)).data.url)).data;
-				})()
-			)
-		);
-	}
+		const { status: reportStatus, identifier: reportId } = (
+			await reportsApi.generateReport(PROJECT_ID, {
+				name: 'top-members',
+				schema: {
+					unit: 'words',
+					format: 'json',
+					dateFrom: '2014-01-01T00:00:00+00:00',
+					dateTo: '2030-01-01T00:00:00+00:00',
```

Comment:
Are we comfortable hard-coding an end date? What if we used either `Date` or `dayjs` to set a date x years in the future?